java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/AccessToken.java

```diff
@@ -36,9 +35,8 @@ public Instant getExpiry() {
         return expiry;
     }
 
-    public long secondsUntilTenMinutesBeforeExpiry(Clock clock) {
-        Instant tenMinutesBeforeExpiry = expiry.minus(10, ChronoUnit.MINUTES);
-        return Duration.between(clock.instant(), tenMinutesBeforeExpiry).toSeconds();
+    public long tenSecondsBeforeExpiry(Clock clock) {
```

Comment:
Does this need to be configurable? I couldn't find any definitive standard for this value

Reply:
We can add configurability later, if we see need for it. Could you keep the `secondsUntil` prefix still? Otherwise an Instant should be returned...

Reply:
Sure, changed. `secondsUntilTenSecondsBeforeExpiry` is not the the most elegant name though